download.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+sem=$1
+
+KORYAVNIKS=(
```

Comment:
Why enumaration?

Reply:
Urls enumeration seems to the easiest way to link urls to sem number, for now I'm going to use separate `.ini` configuration file for it.

Reply:
Agreed.